data/migrations/20201013074025_customer.js

```diff
@@ -2,18 +2,18 @@ exports.up = (knex) => {
   return knex.schema
     .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
     .createTable('Customer', function (table) {
-      table.string('id').notNullable().unique().primary();
-      table.string('description');
-      table.string('name').notNullable();
-      table.string('lastname').notNullable();
-      table.string('address').notNullable();
-      table.string('zip');
-      table.string('phone').notNullable().unique();
-      table.string('email').notNullable().unique();
-      table.string('city');
-      table.string('state');
-      table.string('country');
-      table.string('photo_url').unique();
+      table.integer('id').notNullable().unique().primary();
```

Comment:
converting to int? don't need uuid extension.

Reply:
I'm not yet sure whether I should use uuid instead yet. I'll read about uuid and I'll change id: integer to id: string if that's the case.